rd_ui/app/scripts/directives/query_directives.js

```diff
@@ -58,6 +58,26 @@
     }
   }
 
+  function queryResultExcelLink() {
```

Comment:
We can use the same method for both CSV and XSLX links, by passing the file type as an attribute. But if you are not familiar with Angular enough, leave as is and I will refactor it myself afterwards.